src/main/resources/static/js/status-handler.js

```diff
@@ -0,0 +1,159 @@
+/*
+This js file intend to keep the status information about the current selected back-end
+and depending weather the back end status has changed and or is online or not will
```

Comment:
You are mixing the words "back-end" and "back end",, decide one format.

Reply:
Done.